classes/person.rb

```diff
@@ -0,0 +1,44 @@
+require_relative '../decorators/nameable'
+class Person < Nameable
```

Comment:
- [ ] **[OPTIONAL]** Your Person class does not have the other relationship using `add_rental` as you have in the Book class. I think you should add the same here.

Reply:
Thank you so much and I have implemented the changes